### ISSUE TITLE
False error in CLion for format string

### ISSUE THREAD

**Issue by nandanvasudevan** [intellij-rust/intellij-rust#8412]
## Environment

* **IntelliJ Rust plugin version:** 0.4.162.4321-213
* **Rust toolchain version:** 1.58.0 (02072b482 2022-01-11) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2021.3.2 (CL-213.6461.75)
* **Operating system:** Linux 5.15.8-100.fc34.x86_64
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description
CLion produces an error squiggly for the following line of code which compiles with no errors or warnings.

```rust
    let name: String = "Nandan".to_string();
    println!("{name}");
```
Screenshot of issue error given below.
![image](https://user-images.githubusercontent.com/28172619/149535544-5acdbcf1-03d3-45fd-b96d-4d1f4bfdcbeb.png)

## Steps to reproduce
Try the above code snippet and it should be reproducible.

**Comment by nandanvasudevan** [comment]
Is there anyway to know which version the fix would be available in?

**Comment by Undin** [comment]
It's already available in [nightly](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly).
Stable build with this fix will be available next week

**Comment by nandanvasudevan** [comment]
Thanks!
